core/src/main/java/org/stellar/anchor/util/AssetHelper.java

```diff
@@ -0,0 +1,46 @@
+package org.stellar.anchor.util;
+
+import static org.stellar.anchor.util.StringHelper.isEmpty;
+
+import java.util.Currency;
+import org.stellar.sdk.KeyPair;
+
+public class AssetHelper {
+  public static boolean isISO4217(String assetCode, String assetIssuer) {
+    // assetIssuer must be empty to be a valid Fiat
+    if (!isEmpty(assetIssuer)) {
+      return false;
+    }
+    try {
+      // check if assetCode is ISO4217 compliant
+      Currency.getInstance(assetCode);
+      return true;
+    } catch (Exception ex) {
+      return false;
+    }
+  }
+
+  public static boolean isStellar(String assetCode, String assetIssuer) {
+    if (isEmpty(assetCode)) return false;
+    if (isEmpty(assetIssuer)) return false;
+    // check if assetIssuer is a valid wallet address
+    try {
+      KeyPair.fromAccountId(assetIssuer);
+      return true;
+    } catch (Exception ex) {
+      return false;
+    }
+  }
+
+  public static String getAssetId(String assetCode, String assetIssuer) {
+    if (isISO4217(assetCode, assetIssuer)) {
+      // fiat assets
+      return "iso4217:" + assetCode;
+    } else if (isStellar(assetCode, assetIssuer)) {
+      return "stellar:" + assetCode + ":" + assetIssuer;
+    } else {
+      // not supported
+      return null;
```

Comment:
what about native asset (xlm)?

Reply:
We are not supporting the native assets.